src/rewards/SecretText/SecretTextDataForm/types.ts

```diff
@@ -0,0 +1,5 @@
+export type SecretTextRewardForm = {
```

Comment:
I'm not sure if we need to put these types in a `types.ts` close to the components where we actually use them, or we should just put them in the main `src/types.ts` file. What are the pros and cons in your opinion?

Reply:
Pros:
- The `src/types.ts` won't grow unmaintainably (name collision, too large file, uncertain types)
- The types will be localized in some sense
- This way it's clear that the `types.ts` belongs to `SecretText` from every import, this gives the type additional data for the developers.

Cons:
- More files
- Global types doesn't require imports, as this one does so when something is renamed or moved, more refactor is required usually

I plan to write about these in the project structure convention, this is not something we do in the project now.